File: media-bazaar-website/Classes/Employee.php
<?php

class Employee{

private $Id;
private $firstname;
private $birthdate;
private $gender;
private $married;
private $address;
private $phone;
private $BSN;
private $email;
private $role;
private $salary;


public function __construct($id,$firstname,$birthdate,$gender,$married,$address,$phone,$BSN,$email,$role,$salary){
 $this->Id=$id;
 $this->firstname=$firstname;
 $this->birthdate=$birthdate;
 $this->gender=$gender;
 $this->married=$married;
 $this->address=$address;
 $this->phone=$phone;
 $this->BSN=$BSN;
 $this->email=$email;
 $this->role=$role;
 $this->salary=$salary;

}

public function getFirstName(){
  return $this->firstname;
}

public function getbirthdate(){
  return $this->birthdate;
}

public function getGender(){
  return $this->gender;
}


public function getIsMarried(){
  return $this->married;
}


public function getMarried(){
  if($this->married=='Yes')
    return 1;
  else {
    return 0;
  }
}

public function getgen(){
  if($this->gender=='Male')
    return 1;
  else {
    return 0;
  }
}


public function getAddress(){
  return $this->address;
}

public function getPhone(){
  return $this->phone;
}

public function getBSN(){
  return $this->BSN;
}


public function getEmail(){
  return $this->email;
}

public function getRole(){
  return $this->role;
}

public function getSalary(){
  return $this->salary;
}



public function getID(){
  return $this->Id;
}
public function setID(int $id){
  $this->Id=$id;
}

}

?>

File: Code/Media Bazaar/Media Bazaar/Employees/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.IO;

namespace Media_Bazaar
{
    [Serializable]
    public abstract class Employee 
    {
        protected string username;
        protected string password;
        protected string firstName;
        protected string surname;
        protected int phoneNumber;
        protected string address;
        protected DateTime birthDate;
        protected string email;
        protected int isMarried;
        protected int bsn;
        protected string functions;
        protected int id;
        protected decimal salary;
        protected int gender;
        protected int position;
        protected int? department;
        protected int contract;

        public Employee(string username,string password ,string fName, string surname, int pNumber, string address, DateTime birthDate, string email, int isMarried, int bsn, string functions, int id,decimal salary,int gender,int position,int? department, int contract)
        {
            this.username = username;
            this.password = password;
            this.firstName = fName;
            this.surname = surname;
            this.phoneNumber = pNumber;
            this.address = address;
            this.birthDate = birthDate;
            this.email = email;
            this.isMarried = isMarried;
            this.bsn = bsn;
            this.functions = functions;
            this.id = id;
            this.salary = salary;
            this.gender = gender;
            this.position = position;
            this.department = department;
            this.contract = contract;
        }
        public string GetName()
        {
            return firstName;
        }
      public int  GetPosition()
        {
            return position;
        }
       public int GetId() { return id; }

        public string GetFirstName()
        {
            return firstName;
        }
        public string GetSurname() { return surname; }
        public int GetPhoneNumber() { return phoneNumber; }
        public string GetAddress() { return address; }
        public DateTime GetBirth() { return birthDate; }
        public string GetEmail() { return email; }
        public int GetIsMarried() { return isMarried; }
        public int GetBsn() { return bsn;}
        public string GetFunctions() { return functions; }
       public decimal GetSalary() { return salary; }
        public int GetGender() { return gender; }
        public string GetUsername() { return username; }
        public string GetPassword() { return password; }
        public int? GetDepartment() { return department; }
        public int GetContract() { return contract; }
    }
}
